qgis.dtd

```diff
@@ -22,7 +22,6 @@
 	projectname CDATA #REQUIRED 
 	mapunits (meters|feet|degrees) "meters"
   >
-<!ATTLIST projectlayers layercount CDATA #REQUIRED >
```

Comment:
Is this file a common ressource? Could this change break older versions?

It apparently does not break the loading of old projects for this version of QGIS. And the URL http://mrcc.com/qgis.dtd referenced in the project files produces 404 anyway.